core/import_data/import_ozone_data.py

```diff
@@ -90,6 +115,8 @@ def import_groups():
         "phase_out_year_non_article_5",
     ]
     import_data(Group, settings.IMPORT_RESOURCES_DIR / "groups.json", exclude)
+    # create a group for substances that don't have a group
+    create_uncontrolled_group()
```

Comment:
nu cred ca e nevoie de asta aici, grupul o sa fie creat in import_data

Reply:
dar nu o sa fie creat? in import_data nu se creaza grupul de `other`. se creeaza grupurile din json, apoi la final , in afara `import_data`, se adauga si grupul asta nou.

si mai jos, avand in vedere ca exista optiunea de `import_resources substances`, mai verificam odata daca s-a facut acest grup si daca nu, il adaugam.

Reply:
ah corect corect, ai dreptate